pay-api/migrations/versions/2c1ecdec185a_fas_status_changes.py

```diff
@@ -0,0 +1,39 @@
+"""fas status changes
+
+Revision ID: 2c1ecdec185a
+Revises: f8408a4a782c
+Create Date: 2022-01-05 20:26:15.392781
+
+"""
+from alembic import op
+from sqlalchemy import String
+from sqlalchemy.sql import column, table
+
+# revision identifiers, used by Alembic.
+revision = '2c1ecdec185a'
+down_revision = 'f8408a4a782c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    error_code_table = table('error_codes',
+                             column('code', String),
+                             column('title', String),
+                             column('detail', String)
+                             )
+    op.bulk_insert(
+        error_code_table,
+        [
+            {
+                'code': 'FAS_INVALID_RS_STATUS_CHANGE',
+                'title': 'Routing slip refund is not allowed.',
```

Comment:
Need change to title? Code is generic, but title is specific to refund?

Reply:
fixed